src/test/java/org/fundacionjala/pivotal/steps/story/CreateStorySteps.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacionjala.pivotal.steps.story;
+
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.pivotal.objects.story.Story;
+import org.fundacionjala.pivotal.objects.story.DescriptionsStory;
+import org.testng.Assert;
+
+import java.util.Map;
+
+/**
+ * CreateStorySteps class that create a story.
+ */
+public class CreateStorySteps {
+    private Story createStory;
+    private Map<DescriptionsStory, String> valuesMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "valuesMap" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)